### DISCUSSION TITLE
Is it possible to reduce vertical padding of Collapse?

### DISCUSSION THREAD

**Discussion by xzilja** [heroui-inc/heroui#532] [Q&A] [answered]
Hey guys, I am trying to figure out how to make `Collapse` much smaller. So far I was able to reduce horizontal padding using `css` prop straight on it and made text smaller by using custom element for `title`. However, I can't figure out how to reduce vertical padding, as it is applied by another element that's nested inside `Collapse`. Would appreciate any pointers, thank you.

**Comment by tianenpang** [comment, marked as answer]
Hi @IljaDaderko you can override the style of the elements inside `Collapse` by using the class selector.

Such as `css={{ "& .nextui-something": {...} }}` and here's an example [codesandbox.io/s/collapse-vertical-padding-rxs3wq](https://codesandbox.io/s/collapse-vertical-padding-rxs3wq?file=/pages/index.tsx:638-815).

**Comment by ytkg** [comment]
It would be nice to have an `isCompact` property like `Navbar`.

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2, we added a compact accordion https://nextui.org/docs/components/accordion#compact
